landlab/components/uniform_precip/generate_uniform_precip.py

```diff
@@ -13,7 +13,8 @@
 
 import random
 import numpy as np
-from landlab import Component
+from landlab import Component, ModelGrid
+from six import next
 
 
```

Comment:
A comment less about your changes and more about the documentation. It states in two places that the units are in mm/hr. Shouldn't that be changed to L/T, where T has the same units as the time step and L has the same units as ... whatever is using the generated precipitation (e.g. if precipitation is being used with cell area to calculate an effective, Q, then precipitation should have same L units as grid cells).